src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,16 +1,63 @@
 package wolox.training.controllers;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
```

Comment:
These imports just were deleted automatically. It's weird because GetMapping and RequestParam references are still there, but it seems like they're being imported by another thing (or included in some other import).